cd/deploy.sh

```diff
@@ -40,6 +41,11 @@ if [ "$RELEASE" == 'true' ] && [ "$TRAVIS_BRANCH" == 'master' ] && [ "$TRAVIS_PU
         echo "Tagging version $RELEASE_VERSION"
         echo "--------------------------------------"
         mvn versions:set -DnewVersion=$RELEASE_VERSION -DgenerateBackupPoms=false
+
+        # Updating version properties in core-rest to update SystemProperties
+        sed -i '' 's/^gaffer.version=.*/gaffer.version='$RELEASE_VERSION'/' ../rest-api/core-rest/src/main/resources/version.properties
+        sed -i '' 's/^koryphe.version=.*/koryphe.version='$KORYPHE_POM_VERSION'/' ../rest-api/core-rest/src/main/resources/version.properties
```

Comment:
Please could you do a similar thing just before line 88 ( git commit -a -m "prepare for next development iteration"). You will need to fetch the new versions, then update the version.properties file again with the new versions.